scripts/build/templates/icons-json.d.ts.template

```diff
@@ -0,0 +1,6 @@
+import type {IconData} from '../types.js';
```

Comment:
You probably want to import it from `../types.d.ts` directly.

```suggestion
import type {IconData} from '../types.d.ts';
```

Reply:
See our `sdk.d.ts` file, it's using `types.js`. I just used the same format as our existing file.